Framework/Algorithms/src/NormaliseByCurrent.cpp

```diff
@@ -57,6 +57,8 @@ double NormaliseByCurrent::extractCharge(
                            "wrong current.\n";
   }
   // Handle multiperiod data.
+  // The number of periods is set above by reference
```

Comment:
As this is at the heart of PropertyWithValue doing so in this ticket would add considerable risk and expand the scope sgnificantly.